unmanic/libs/ffmpeg.py

```diff
@@ -308,8 +307,37 @@ def check_file_to_be_processed(self, vid_file_path, settings):
         else:
             correct_video_codec = True
 
+        correct_audio_codec = False
+        if settings['enable_audio_encoding']:
+            try:
+                audio_streams_codecs = ""
+                for stream in file_probe['streams']:
+                    if stream['codec_type'] == 'audio':
+                        # Check if this file is already the right format
+                        audio_streams_codecs += "{},{}".format(audio_streams_codecs, stream['codec_name'])
+                        if stream['codec_name'] == settings['audio_codec']:
+                            if settings['debugging']:
+                                self._log("File already has {} codec audio stream - {}".format(settings['audio_codec'], vid_file_path),
```

Comment:
I'll merge this and change it myself, but the settings['debugging'] condition here is pointless (also below).
If the log entry level is set as `level='debug'` as these logs are, then the logger is already setup to only log them if debugging is set in settings.